### ISSUE TITLE
conversão problemática PDF->DOCX->XML

### ISSUE THREAD

**Issue by odanoburu** [own-pt/oab-exams#23]
erros identificados:

código de ética da OAB, artigos: 24, 31

regulamento geral da OAB, artigos: 88,

**Comment by odanoburu** [comment]
1. vocês acham que tentamos consertar na fonte DOCX ou consertamos no XML direto? 

2. é prioridade no momento?

**Comment by arademaker** [comment]
@odanoburu depende dos errors e como eles influenciam o resultado final. Seria legal descrever os erros para não esquecermos.

**Comment by odanoburu** [comment]
os erros são do tipo:

```xml
<Artigo id="art31">
<Rotulo>Art. 31.</Rotulo>
<Caput id="art31_cpt">
<p>
O anúncio não deve conter fotografias, ilustrações, cores, figuras, desenhos, logotipos, marcas ou símbolos incompatíveis com a sobriedade da advocacia, sendo proibido o uso dos símbolos oficiais e dos que sejam utilizados pela Ordem dos Advogados do Brasil.
</p>
</Caput>
</Artigo>
<p>
<li>
1º São vedadas referências a valores dos serviços, tabelas, gratuidade ou forma de pagamento, termos ou expressões que possam iludir ou confundir o público, informações de serviços jurídicos suscetíveis de implicar, direta ou indiretamente, captação de causa ou clientes, bem como menção ao tamanho, qualidade e estrutura da sede profissional.
</li>
</p>
<p>
<li>
2º Considera-se imoderado o anúncio profissional do advogado mediante remessa de correspondência a uma coletividade, salvo para comunicar a clientes e colegas a instalação ou mudança de endereço, a indicação expressa do seu nome e escritório em partes externas de veículo, ou a inserção de seu nome em anúncio relativo a outras atividades não advocatícias, faça delas parte ou não.
</li>
</p>
```

i.e., o elemento Artigo é fechado antes do texto do artigo de fato terminar.

**Comment by arademaker** [comment]
Em b185991 corrigi este caso específico. Acho que precisamos resolver né? Estes casos vão impactar diretamente o experimento, certo?

**Comment by odanoburu** [comment]
ok, vou fazer isso

**Comment by arademaker** [comment]
Acabei outro commit ece537f mas alguns dos erros apontados já tinham sido corrigidos.

**Comment by odanoburu** [comment]
uma coisa: esse commit ece537f08be8 corrige um problema que na verdade são as notas de rodapé do arquivo. como lidamos com isso na lei? @pdelfino isso deve ser removido? (talvez seja melhor remover na fonte DOCX...)

**Comment by arademaker** [comment]
@odanoburu rodapé? Bem, sobre corrigir nos DOCX, sim esta deveria ser a forma certa, estou mexendo no XML direto só para não atrapalhar o experimento. 6 artigos com problema podem impactar os experimentos né? Por isso sugeri corrigir.

**Comment by odanoburu** [comment]
sim, tipo essas: ![image](https://user-images.githubusercontent.com/19347395/30645591-93c0cf5e-9dec-11e7-9459-f2d8276bce72.png)

oq vc acha melhor pra agora? corrige no DOCX e depois no XML (o que não tem a ver com rodapé) ou corrige tudo no XML de uma vez?

**Comment by odanoburu** [comment]
tem errinhos tb do tipo alguns artigos estarem fora de seções, mas não altera nada para os nossos fins.

**Comment by odanoburu** [comment]
ver #28.

acho que vamos ter problemas se precisarmos de usar muitas normas que o parser não entenda..

se quisermos descer o nível de artigo para parágrafo já não dá, por causa dos erros.
